lib/members_page.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class MembersPage < Scraped::HTML
+  decorator Scraped::Response::Decorator::CleanUrls
+
+  field :member_urls do
+    noko.css('div#RESULTADOS_DIPUTADOS div.listado_1 ul li a').map { |p| p[:href] }
```

Comment:
As per the comments on https://github.com/everypolitician-scrapers/spain_congreso_es/pull/17, our normal approach to this is to have the `a/@href` in the selector. If there's a reason why that won't work here, you should note it, otherwise it's better to be consistent.